validator/sawtooth_validator/networking/dispatch.py

```diff
@@ -264,6 +264,11 @@ def _determine_next(self, message_id, future):
                     correlation_id=original_message.correlation_id,
                     message_type=res.message_type)
                 try:
+                    LOGGER.warning("Sending hang-up in reply to %s to "
```

Comment:
Can this message be on a single line? It's hard to grep for messages when they're broken up.